src/components/VModal/VModalContent.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <VTeleport v-if="visible" to="modal">
+    <div
+      class="flex justify-start items-center flex-col z-10 inset-0 overflow-y-auto fixed inset-0 bg-dark-charcoal bg-opacity-75 transition-opacity min-h-screen"
+      v-bind="$attrs"
+      aria-modal="true"
+      role="dialog"
+      v-on="$listeners"
+      @keydown="onKeyDown"
+      @blur="onBlur"
+    >
+      <div class="w-4/5 flex justify-end">
+        <VButton
+          size="disabled"
+          variant="floating"
+          class="text-white py-2"
```

Comment:
```suggestion
          class="text-white my-2"
```

The focus style is less than ideal with this as padding:

<img width="125" alt="CleanShot 2021-12-02 at 10 36 32@2x" src="https://user-images.githubusercontent.com/6351754/144453434-f6234bc0-1b57-40ce-a0e9-70f498aa0d23.png">

After switching to margin:

<img width="105" alt="CleanShot 2021-12-02 at 10 36 21@2x" src="https://user-images.githubusercontent.com/6351754/144453477-e80950ab-af29-48bb-9390-2ad00f07a7e4.png">

Alternatively, `py-1 my-1` produces an in=between result:

<img width="147" alt="CleanShot 2021-12-02 at 10 38 22@2x" src="https://user-images.githubusercontent.com/6351754/144453637-8b6f62a6-9768-4f8c-8376-3494d8cb64f0.png">